Sources/TestingMacros/TestDeclarationMacro.swift

```diff
@@ -527,13 +533,33 @@ public struct TestDeclarationMacro: PeerMacro, Sendable {
       @frozen enum \(enumName): Testing.__TestContainer {
         static var __tests: [Testing.Test] {
           get async {
-            \(raw: testsBody)
+            [
+              await \(testGetterFunctionName)()
+            ]
           }
         }
       }
       """
     )
 
+    // DISABLED: Global variable must be a compile-time constant to use @_section attribute
+//    let exportedSymbolName = context.makeUniqueName(thunking: functionDecl)
+//    result.append(
+//      """
+//      #if hasFeature(SymbolLinkageMarkers)
+//      @_used
+//        #if os(macOS) || os(iOS) || os(watchOS) || os(tvOS) || os(visionOS)
+//        @_section("__DATA_CONST,__swift5_tests")
+//        #elseif os(Linux)
+//        @_section("swift5_tests")
+//        #elseif os(Windows)
+//        @_section(".sw5test")
```

Comment:
```suggestion
//        @_section("sw5test$B")
```

Reply:
@kubamracek Do you know if `@_section` requires the leading period or not?

Reply:
Okay, I think I got it right then.